_episodes/01-shell-basics.md

```diff
@@ -8,42 +8,43 @@ questions:
 objectives:
 - "Understand basic differences among available shells and their use in
  neuroimaging toolkits"
-- "Using shell as a working medium explain the role of some most important environment variables"
+- "Using a shell as a working medium"
+- "Explain the role of some of the most important environment variables"
 - "Provide hints on efficient use of the collected shell history of commands"
 - "Explain how to make shell scripts more robust and less dangerous"
-- "Introduce basics of runtime- and unit- testing"
+- "Introduce basics of runtime and unit testing"
 keypoints:
-- "There are different (incompatible) shells and different
-  neuroimaging tools might use a different shell and thus provide
-  instructions which would not be compatible with your current shell"
-- "Command line shell is a powerful tool, and learning additional
+- "There are a number of incompatible shells. Different
+  neuroimaging tools may use specific shells and thus provide
+  instructions that are not compatible with your current shell."
+- "A command line shell is a powerful tool and learning additional
  'tricks' could help to make its use much more efficient, less
- error-prone, and thus more reproducible"
+ error-prone, and thus more reproducible."
```

Comment:
as you might have noted, me is "greedy" (and somewhat consistent) on trailing periods if some other separator is used and it is clear that statement is 'done'.  That is why in this case, as well as among Objectives, I did not use trailing period.  In general -- I do not care much but OCD-me likes to be consistent